src/main/webapp/i18n/de/quizExercise.json

```diff
@@ -160,6 +160,32 @@
                 "proportional_with_penalty": "Anteilig mit Abzügen"
             },
             "importJSON": "Fragen importieren (JSON)",
+            "explanationText":{
+                "point":"Punkt",
+                "points":"Punkte",
+                "item":"Element",
+                "items":"Elemente",
+                "textgap":"Lücke",
+                "textgaps":"Lücken",
+                "option":"Antwortmöglichkeit",
+                "options":"Antwortmöglichkeiten",
+                "scoringTitle":"Erklärung zur Punkteverteilung in Frage {{paramIndex}}) {{paramTitle}}",
+                "scoringTypeProportional":"Verteilung Anteilig mit Abzügen: ",
+                "scoringTypeAllOrNothing":"Verteilung Alles-oder-Nichts: ",
+                "scoringTypeProportionalExplanation": "Jede richtige Antwort gibt positive Punkte, jeder Fehler gibt negative Punkte, um Rätselraten zu vermeiden. ",
+                "scoringTypeAllOrNothingExplanation": "Sie erhalten entweder alle oder gar keine Punkte..",
+                "multipleChoiceAllOrNothingCorrect": "Sie haben {{paramScore}} {{paramPoint}} der {{paramQuestionScore}} möglichen {{paramScorePoint}} erreicht. Da Sie alle Antwortoptionen richtig ausgewählt haben, haben Sie in dieser Übung alle Punkte erreicht. Tolle Arbeit!",
```

Comment:
I would generally say "du" instead of "Sie". We are at the university, and "du" is more personal